martin/src/args/root.rs

```diff
@@ -107,6 +107,11 @@ impl Args {
             config.mbtiles = parse_file_args(&mut cli_strings, "mbtiles", false);
         }
 
+        #[cfg(feature = "cog")]
+        if !cli_strings.is_empty() {
+            config.cog = parse_file_args(&mut cli_strings, "tif", false);
```

Comment:
so we decided not to support multi-extension for now? (i.e. `*.tiff`)    I'm ok with it, as long as its a conscious choice.

Reply:
Has been added to plan and tracked in [#875](https://github.com/maplibre/martin/issues/875)